vars/runDirectDebitE2E.groovy

```diff
@@ -3,13 +3,18 @@
 def call(
         String app = null,
         String tag = null,
-        String pay_scripts_branch = 'master') {
+        String pay_scripts_branch = null) {
+
+    commit = env.GIT_COMMIT ?: gitCommit()
 
     if (tag == null) {
-        commit = env.GIT_COMMIT ?: gitCommit()
         tag = "${commit}-${env.BUILD_NUMBER}"
     }
 
+    if (app == 'scripts' && pay_scripts_branch == null) {
+        pay_scripts_branch = commit
+    }
+
```

Comment:
wrap up this logic in a function maybes?

Reply:
I like that is explicit, I don't think it'd be as clear if it were extracted to a function